source/MultiGrid/MultiGrid.js

```diff
@@ -99,15 +100,15 @@ export default class MultiGrid extends Component {
     const { columnWidth, fixedColumnCount, fixedRowCount, rowHeight } = this.props
 
     if (
-      columnWidth !== nextProps.columnWidth ||
+      columnWidth instanceof Function || columnWidth !== nextProps.columnWidth ||
       fixedColumnCount !== nextProps.fixedColumnCount
     ) {
       this._leftGridWidth = null
     }
 
     if (
       fixedRowCount !== nextProps.fixedRowCount ||
-      rowHeight !== nextProps.rowHeight
+      rowHeight instanceof Function || rowHeight !== nextProps.rowHeight
```

Comment:
That makes sense on the optimization. By what method should the cache be cleared?
Perhaps a new public method like:
```
clearCachedStyles() {
  this._topGridHeight = null;
  this._leftGridWidth = null;
  this._maybeCalculateCachedStyles(null, this.props)
}
```